tests/terraform_main_test.go

```diff
@@ -0,0 +1,33 @@
+package tests
+
+import (
+	"os"
+	"testing"
+
+	"github.com/gruntwork-io/terratest/modules/terraform"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestTerraformEksCluster(t *testing.T) {
+	// Construct the terraform options with default retryable errors to handle the most common
+	// retryable errors in terraform testing.
+	terraformOptions := terraform.WithDefaultRetryableErrors(t, &terraform.Options{
+		// Set the path to the Terraform code that will be tested.
+		TerraformDir: "../",
+	})
+
+	// Clean up resources with "terraform destroy" at the end of the test.
+	if os.Getenv("DISABLE_DESTROY") != "" {
+		defer terraform.Destroy(t, terraformOptions)
+	}
+
+	// Select the "default" workspace for running tests
+	terraform.WorkspaceSelectOrNew(t, terraformOptions, "default")
+
+	// Run "terraform init" and "terraform apply". Fail the test if there are any errors.
+	terraform.InitAndApply(t, terraformOptions)
+
+	// Run `terraform output` to get the values of output variables and check they have the expected values.
+	output := terraform.Output(t, terraformOptions, "cluster_id")
+	assert.NotEqual(t, "", output)
```

Comment:
Consider renaming a variable for clarity
```suggestion
	tf_output_cluster_id := terraform.Output(t, terraformOptions, "cluster_id")
	assert.NotEqual(t, "", tf_output_cluster_id)
```